ci/multibuild_config.sh

```diff
@@ -2,5 +2,5 @@
 # Test for OSX with [ -n "$IS_OSX" ]
 
 function run_tests {
-    tox
+    pytest
```

Comment:
Forgot to mention (though it’s in multibuild docs) that this function runs from within a new temporary subfolder, so if you want to run pytest from the root of th repo, you need to cd ..

Reply:
https://github.com/fonttools/ttfautohint-py/blob/f099836e367bbc0d8f6652ac23f8e849d5b366b6/ci/multibuild_config.sh#L45